src/component/Generation/Generation.js

```diff
@@ -19,6 +19,7 @@ export default () => {
   } = useCharacter();
   const useContract = useContext(ContractContext);
   const { creating } = useContract;
```

Comment:
create and creating can be fetching in same line:
`const {creating, create} =useContract`